content/topics/ENA-submission-tutorial.md

```diff
@@ -0,0 +1,473 @@
+    </button>
+</div>
+<pre><code>
+java -jar ~/webin-cli-8.2.0.jar -ascp -context genome -userName Webin-XXXXX -password 'YYYYY' -manifest ./Alectoris-graeca-manifest.txt -validate -test
+<br></code></pre>
+<div class="clipboard">
+    <button class="clipboard-button copy-button" type="button">
+        <img class="clipboard-icon" src="/img/icons/copy.svg" alt="Copy to clipboard icon">
+    </button>
+    <button class="clipboard-button copied-button" type="button" style="display: none;">
+        <img class="clipboard-icon" src="/img/icons/check.svg" alt="Copy to clipboard icon">
+    </button>
+</div>
+<pre><code>
+java -jar ~/webin-cli-8.2.0.jar -ascp -context genome -userName Webin-XXXXX -password 'YYYYY' -manifest ./Alectoris-graeca-manifest.txt -submit -test
+</code></pre>
+            </li>
+            <li><b>Note:</b> The <code>-test</code> indicates that the submission will be made to the test instance. Omit this when doing real submissions.</li>
+            <li> Upon a successful submission, an accession starting with ERZ will be obtained. However, this is not the official accession to be used in a publication. ENA will need to process the assembly, and when this is done you will receive an email with the genome assembly accession to be used (GCA_xxxxxx), e.g.: 
+<pre><code>
+    ASSEMBLY_NAME | ASSEMBLY_ACC  | STUDY_ID   | SAMPLE_ID   | CONTIG_ACC                      | SCAFFOLD_ACC | CHROMOSOME_ACC
+    bAleGra1.1    | GCA_965278835 | PRJEB79727 | ERS17759205 | CBDIHD010000001-CBDIHD010000076 |              | OZ257071-OZ257110
+<br></code></pre>
+            </li>
+        </ul>
+    </span>
+  </div>  
+  <br>
+</div>
+
+### Programmatic Submission Method
+This method requires the preparation of XML files that are sent to ENA using cURL (or uploading them via the Webin Portal of ENA). It is possible to register and submit **studies**, **samples**, and **raw reads** this way. This is the **only** way to register **umbrella** studies.
+
+Any programmatic submission requires two .xml files, one with the action (ADD, MODIFY, RELEASE, CANCEL) and one with the metadata. The submission is done via a cURL command in a terminal window, and all levels (study, sample, raw reads) can be registered and submitted at the same time, but in this tutorial the levels will be submitted separately.
+
+#### Test submission
+ENA provides two sites for submission; one for test submissions (`https://wwwdev.ebi.ac.uk/ena/submit/drop-box/submit/`) and one for 'real' submissions (i.e. an actual submission to ENA) (`https://www.ebi.ac.uk/ena/submit/drop-box/submit/`). We recommend doing a test submission first, using the example data provided, in order to get an understanding of the different steps. <b>Note:</b> The test service is purged every 24 h, meaning that any submitted test file will be removed before the following day, making it impossible to begin a submission one day and continue the next.
```

Comment:
Replace 'real' with production: "... and one for production submissions (i.e. an actual submission to ENA)"